arangod/Aql/AstNode.cpp

```diff
@@ -816,27 +816,28 @@ uint64_t AstNode::hashValue(uint64_t hash) const noexcept {
 
 /// @brief dump the node (for debugging purposes)
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
-void AstNode::dump(int level) const {
```

Comment:
Shouldn't this be `operator<<`?

Reply:
I am not sure how to pass the indent to the operator.